packages/@aws-cdk/aws-synthetics/lib/canary.ts

```diff
@@ -77,6 +77,14 @@ export class Runtime {
    */
   public static readonly SYNTHETICS_1_0 = new Runtime('syn-1.0');
 
+  /**
+   * `syn-nodejs-2.0` includes the following:
+   * - Lambda runtime Node.js 10.x
+   * - Puppeteer-core version 3.3.0
+   * - Chromium version 81.0.4044.0
+   */
+  public static readonly SYNTHETICS_NODEJS_2_0 = new Runtime('syn-nodejs-2.0');
```

Comment:
probably also worth an
```
@see https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries_Library.html#CloudWatch_Synthetics_runtimeversion-2.0`
```
to help users make the decision on which runtime fits their use case.

should we also update this [doc string](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries_Library.html#CloudWatch_Synthetics_runtimeversion-2.0)

Reply:
> should we also update this [doc string](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries_Library.html#CloudWatch_Synthetics_runtimeversion-2.0)

which one?

Also, what about changing the default runtime?

Reply:
oops, bad link from me - [doc string](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-synthetics/lib/canary.ts#L183)

I suppose it is experimental so we *could* still update the default runtime if it's the sensible thing we want to nudge users towards. thoughts @NetaNir ?

Reply:
Yeah I think that since we are still in the experimental phase it make sense to change it